### PR TITLE
add storage stats and contracts related fields

### PR DESCRIPTION
**Pull request by wxue**
ref: [BTFS Node API Design(Appendix) - UI/UX Related](https://docs.google.com/document/d/1OYMX4T-s_UexLooyo4Ldhj0rpN8Mh9DvurenLFTy4Zg/edit#heading=h.1me4yqc0bp9l)

### CODE REVIEW

File: protos/node/node.proto [hunk 1]
```diff
@@ -90,3 +90,46 @@ message Node {
...
+message StorageStat {
+  message Host {
+    bool online = 1;
+    float uptime = 2;
+    float score = 3;
+    int64 storage_used = 4;
+    int64 storage_cap = 5;
+  }
+  message Renter {
+    string reversed = 1;
```
Comment (taiyangc) [inline protos/node/node.proto]: `reserved`?

File: protos/node/node.proto [hunk 2]
```diff
@@ -90,3 +90,46 @@ message Node {
...
+  }
+  Host host_stats = 1 [(gogoproto.nullable) = false];
+  Renter renter_stats = 2 [(gogoproto.nullable) = false];
+}
+
+message ContractStat {
+  int64 active_contract_num = 1;
+  int64 compensation_paid = 2;
+  int64 compensation_outstanding = 3;
+  string first_contract_start = 4;
```
Comment (taiyangc) [inline protos/node/node.proto]: shouldn't this be a timestamp?

File: protos/node/node.proto [hunk 3]
```diff
@@ -90,3 +90,46 @@ message Node {
...
+  string last_contract_end = 5;
```
Comment (taiyangc) [inline protos/node/node.proto]: timestamp?

File: protos/node/node.proto [hunk 4]
```diff
@@ -90,3 +90,46 @@ message Node {
...
+  string role = 6;
```
Comment (taiyangc) [inline protos/node/node.proto]: define enum?

File: protos/node/node.proto [hunk 5]
```diff
@@ -90,3 +90,46 @@ message Node {
...
+}
+
+message Contracts {
+  message Contract {
+    string contract_id = 1;
+    string host_id = 2;
+    string renter_id = 3;
+    string status = 4;
```
Comment (taiyangc) [inline protos/node/node.proto]: can you import maybe escrowpb and use the status defined there?

File: protos/node/node.proto [hunk 6]
```diff
@@ -90,3 +90,46 @@ message Node {
...
+    string start_time = 5;
+    string end_time = 6;
+    string next_escrow_time = 7;
```
Comment (taiyangc) [inline protos/node/node.proto]: timestamps?

**Comment by wxue** [comment]
@taiyangc Good catch! Thanks for the comments. Updated and ready for another review.

**Pull request merged**